CHANGELOG.md

```diff
@@ -42,6 +42,7 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
   - The permission `Compliance` will deprecate the permission `ComplianceRuns`.
 
 ### Technical Changes
+- ROX-11268: An API to report which processes are listening to which ports has been added. The API is at `/v1/processes_listening_on_ports`
```

Comment:
I don't believe this is a candidate for changelog

Reply:
Okay. I can remove it. I thought that new APIs should be documented in the changelog. Is the authomatically genrated swagger document enough?

Reply:
New features are announced through release notes. Changelog is meant to point out breaking changes to existing functionality.